manager/integration/tests/test_ha.py

```diff
@@ -855,16 +845,11 @@ def test_rebuild_with_inc_restoration(
 
     # cleanup
     backupstore_cleanup(client)
-    delete_and_wait_pod(core_api, std_pod_name)
-    delete_and_wait_pvc(core_api, std_pvc_name)
-    delete_and_wait_pv(core_api, std_pv_name)
-    delete_and_wait_pod(core_api, dr_pod_name)
-    delete_and_wait_pvc(core_api, dr_pvc_name)
```

Comment:
Same above.

Reply:
In this case, the pod is getting created with pod_make fixture and as we have PV/PVC cleanup there. These delete methods are not required.